core/domain/classifier_services.py

```diff
@@ -442,7 +445,17 @@ def load_examples(self, examples):
             xrange(self._num_docs))
 
     def predict_label_for_doc(self, d):
-        """Returns the predicted label from a doc's prediction report."""
+        """Returns the predicted label from a doc's prediction report.
+
+        Classifiers built with less than _DEFAULT_MIN_DOCS_TO_PREDICT will
+        likely not be useful for predicting, and could potentially cause
+        more issues with incorrect predictions. Because prediction uses
+        Prob(the doc should be assigned this label | the doc is not assigned
+        DEFAULT_LABEL), if there are only two labels (the default label and
+        one other) then the one other label will always be predicted."""
```

Comment:
This comment has been addressed after the larger portion of the docstring was removed.